src/nft/components/explore/CarouselCard.tsx

```diff
@@ -81,16 +81,15 @@ const CardHeaderColumn = styled.div`
   align-items: center;
   flex-direction: column;
   gap: 8px;
-  padding: 40px;
+  height: 202px;
+  justify-content: center;
+  padding: 0 40px;
```

Comment:
Unrelated but can we reduce the 40px padding on mobile screens? Names are truncating seemingly early from the lost space

Reply:
Yes, I think we could reduce it to 20px on smaller devices?

Reply:
I'll do that in a follow-up when I am working on responsive tweaks